android/app/src/main/java/com/microsoft/codepush/common/utils/FileUtils.java

```diff
@@ -137,27 +135,26 @@ public void deleteDirectoryAtPath(String directoryPath) throws IOException {
      * @param file path to file/folder to be deleted.
      */
     @SuppressWarnings("WeakerAccess")
-    public void deleteFileOrFolderSilently(File file) {
+    public void deleteFileOrFolderSilently(File file) throws IOException {
```

Comment:
This method should't throw any errors as said in his description , so please remove this change and place TODO comment here in order to don't forget to return here later to add `trackException` instead.